src/amo/components/SearchFilters/index.js

```diff
@@ -245,6 +289,51 @@ export class SearchFiltersBase extends React.Component<InternalProps> {
             </div>
           )}
 
+          {filters.addonType === ADDON_TYPE_STATIC_THEME && (
+            <div>
+              <label
+                className="SearchFilters-label SearchFilters-Color-label"
+                htmlFor="SearchFilters-CustomColor"
+              >
+                {i18n.gettext('Theme Color')}
+              </label>
+              <ul className="SearchFilters-ThemeColors">
```

Comment:
Why is this not a "<select>" element? Seems odd..

Reply:
A "" element ?

Reply:
Github rendering.... a "select" element with "option" options. I see now why though. You're presenting the options as independent clickable buttons.

Are elements selectable via keyboard. I know we had discussed previously ensuring they are accessible for color accessibility but what about keyboard navigation/screen reader accessibility?

Is there a reason not to use button elements? or even radio?

Reply:
They are not currently accessible through keyboard, you're right. I can't use `<input type=radio>` because there is no way to unselect radio buttons. I could use individual `<button>` elements though.

Reply:
7b1d3c1cd2e4b5f9086171914faa50413cc0e1a1 fixes that, making them `<button>` with a tooltip and a visually hidden label for screen readers (similar to how we do the star rating button)